guides/views.py

```diff
@@ -17,6 +17,19 @@ class IndexView(generic.ListView):
     paginate_by = 10
 
 
+class SearchView(generic.ListView):
+    context_object_name = 'latest_guides'
+    model = Guide
+
+    def get_queryset(self):
+        term = self.request.GET.get('term')
+        if term.lower() == 'all':
```

Comment:
Is this for an unspecified search query, or for when the user literally specifies `all`? You should be checking for `if not term` here instead of `term.lower() == 'all'`. If the user decides to remove the `term` kwarg from the URL for whatever reason, `self.request.GET.get('term')` returns `None`, and then you have an `AttributeError` here.

Reply:
Thank you for pointing out!